src/nvim/getchar.c

```diff
@@ -1494,6 +1496,12 @@ int vgetc(void)
    */
   may_garbage_collect = FALSE;
 
+  // exit repeat and retry
+  if (c == K_REPEAT_END) {
+    repeat_busy = 0;
+    c = K_IGNORE;
```

Comment:
I'd like someone to give me a suggestion on what we shall return here.
I tried `K_REPEAT_END` itself and `NUL` but both were harmful.

Reply:
what's wrong with `K_IGNORE` ?